src/libcore/ops.rs

```diff
@@ -102,6 +102,9 @@ pub trait Drop {
     ///
     /// After this function is over, the memory of `self` will be deallocated.
     ///
+    /// This function cannot be called explicitly. This is compiler error
+    /// [0040](https://doc.rust-lang.org/error-index.html#E0040).
```

Comment:
Because the destructor does not take ownership. There’s a ton of book-keeping minutiæ going behind the covers other than just calling the destructor when a type is dropped too. Simply calling the `Drop::drop` wouldn’t really drop the type, only run the destructor.